Source/WebCore/editing/cocoa/DictionaryLookup.mm

```diff
@@ -411,8 +411,14 @@ static void expandSelectionByCharacters(PDFSelection *selection, NSInteger numbe
     [selection extendSelectionAtStart:lookupAddedBefore];
     [selection extendSelectionAtEnd:lookupAddedAfter];
 
-    ASSERT([selection.string isEqualToString:[fullPlainTextString substringWithRange:extractedRange]]);
-    return selection.string;
+    NSString *selectionString = selection.string;
+
+    auto extractedStringMatchesSelection = [](auto selection, auto extracted) -> bool {
```

Comment:
this can probably be guarded by ASSERT_ENABLED so that it's not unnecessary performed

Reply:
Right